2017-03-13 11:47:53,893 [Dummy-1:140333140121344] [django.db.backends:90] [utils:execute] [DEBUG]- (0.007) 
            SELECT name, type FROM sqlite_master
            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'
            ORDER BY name; args=None
2017-03-13 11:47:53,924 [Dummy-1:140333140121344] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
2017-03-13 11:49:18,385 [Dummy-1:140693114271488] [django.db.backends:90] [utils:execute] [DEBUG]- (0.001) 
            SELECT name, type FROM sqlite_master
            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'
            ORDER BY name; args=None
2017-03-13 11:49:18,392 [Dummy-1:140693114271488] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
2017-03-13 11:58:38,458 [Dummy-1:139756241876736] [django.db.backends:90] [utils:execute] [DEBUG]- (0.001) 
            SELECT name, type FROM sqlite_master
            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'
            ORDER BY name; args=None
2017-03-13 11:58:38,461 [Dummy-1:139756241876736] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
2017-03-13 11:58:57,000 [Dummy-1:140200331323136] [django.db.backends:90] [utils:execute] [DEBUG]- (0.001) 
            SELECT name, type FROM sqlite_master
            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'
            ORDER BY name; args=None
2017-03-13 11:58:57,004 [Dummy-1:140200331323136] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
2017-03-13 12:07:03,300 [Dummy-1:140271328139008] [django.db.backends:90] [utils:execute] [DEBUG]- (0.001) 
            SELECT name, type FROM sqlite_master
            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'
            ORDER BY name; args=None
2017-03-13 12:07:03,318 [Dummy-1:140271328139008] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
2017-03-13 12:11:07,664 [Dummy-1:139666059921152] [django.db.backends:90] [utils:execute] [DEBUG]- (0.001) 
            SELECT name, type FROM sqlite_master
            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'
            ORDER BY name; args=None
2017-03-13 12:11:07,667 [Dummy-1:139666059921152] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
2017-03-13 12:21:53,492 [Dummy-1:139807021397760] [django.db.backends:90] [utils:execute] [DEBUG]- (0.001) 
            SELECT name, type FROM sqlite_master
            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'
            ORDER BY name; args=None
2017-03-13 12:21:53,496 [Dummy-1:139807021397760] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
2017-03-13 12:22:16,193 [Thread-2:139807009523456] [django.template:929] [base:_resolve_lookup] [DEBUG]- Exception while resolving variable 'SITE_DESC' in template 'index.html'.
Traceback (most recent call last):
  File "/home/cc/project_virtualenv/blog_project_venv/local/lib/python2.7/site-packages/django/template/base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [SITE_DESC] in u"[{'False': False, 'None': None, 'True': True}, {u'csrf_token': <SimpleLazyObject: <function _get_val at 0x7f275a9fcb18>>, 'SITE_NAME': '\\xe6\\x92\\xb8\\xe8\\xae\\xaf\\xe5\\x85\\x88\\xe7\\x94\\x9f', 'user': <SimpleLazyObject: <function <lambda> at 0x7f275b455500>>, 'SITE_DECS': '\\xe4\\xb8\\x96\\xe7\\x95\\x8c\\xe4\\xb8\\x8a\\xe6\\x9c\\xac\\xe6\\xb2\\xa1\\xe6\\x9c\\x89\\xe5\\x9f\\xba\\xe4\\xbd\\xac\\xef\\xbc\\x8c\\xe8\\xaf\\xb4\\xe7\\x9a\\x84\\xe4\\xba\\xba\\xe5\\xa4\\x9a\\xe4\\xba\\x86\\xef\\xbc\\x8c\\xe4\\xb9\\x9f\\xe5\\xb0\\xb1\\xe6\\x88\\x90\\xe4\\xba\\x86\\xe5\\x9f\\xba\\xe4\\xbd\\xac', 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x7f275aa0e750>, 'DEFAULT_MESSAGE_LEVELS': {'DEBUG': 10, 'INFO': 20, 'WARNING': 30, 'SUCCESS': 25, 'ERROR': 40}, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x7f275b453950>, u'request': <WSGIRequest: GET '/'>}, {}, {'request': <WSGIRequest: GET '/'>}]"
2017-03-13 12:22:16,219 [Thread-2:139807009523456] [django.server:131] [basehttp:log_message] [INFO]- "GET / HTTP/1.1" 200 14898
2017-03-13 12:22:16,441 [Thread-3:139807009523456] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/css/base.css HTTP/1.1" 304 0
2017-03-13 12:22:16,442 [Thread-4:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/css/index.css HTTP/1.1" 304 0
2017-03-13 12:22:16,460 [Thread-5:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/js/jquery.min.js HTTP/1.1" 304 0
2017-03-13 12:22:16,467 [Thread-6:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/js/sliders.js HTTP/1.1" 304 0
2017-03-13 12:22:16,530 [Thread-7:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/js/nav.js HTTP/1.1" 304 0
2017-03-13 12:22:16,614 [Thread-8:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_06_1.jpg HTTP/1.1" 304 0
2017-03-13 12:22:16,618 [Thread-9:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/a1.jpg HTTP/1.1" 304 0
2017-03-13 12:22:16,621 [Thread-10:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_04_1.jpg HTTP/1.1" 304 0
2017-03-13 12:22:16,625 [Thread-11:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_07_1.jpg HTTP/1.1" 304 0
2017-03-13 12:22:16,645 [Thread-12:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ph2.png HTTP/1.1" 304 0
2017-03-13 12:22:16,654 [Thread-13:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/datepng.png HTTP/1.1" 304 0
2017-03-13 12:22:16,669 [Thread-14:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/icons.png HTTP/1.1" 304 0
2017-03-13 12:22:16,674 [Thread-15:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/a2.jpg HTTP/1.1" 304 0
2017-03-13 12:22:16,688 [Thread-16:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/a3.jpg HTTP/1.1" 304 0
2017-03-13 12:22:16,690 [Thread-17:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/a4.jpg HTTP/1.1" 304 0
2017-03-13 12:22:16,702 [Thread-18:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/newsbg01.png HTTP/1.1" 304 0
2017-03-13 12:22:16,726 [Thread-19:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/slide-bg.png HTTP/1.1" 304 0
2017-03-13 12:22:16,730 [Thread-22:139806992213760] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/silde-nav.png HTTP/1.1" 304 0
2017-03-13 12:22:16,732 [Thread-21:139807009523456] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/newsbg02.png HTTP/1.1" 304 0
2017-03-13 12:22:16,746 [Thread-20:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/newsbg04.png HTTP/1.1" 304 0
2017-03-13 12:22:16,750 [Thread-23:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/newsbg03.png HTTP/1.1" 304 0
2017-03-13 12:22:16,755 [Thread-24:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_05_1.jpg HTTP/1.1" 304 0
2017-03-13 12:22:46,645 [Thread-27:139807000868608] [django.template:929] [base:_resolve_lookup] [DEBUG]- Exception while resolving variable 'SITE_DESC' in template 'index.html'.
Traceback (most recent call last):
  File "/home/cc/project_virtualenv/blog_project_venv/local/lib/python2.7/site-packages/django/template/base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [SITE_DESC] in u"[{'False': False, 'None': None, 'True': True}, {u'csrf_token': <SimpleLazyObject: <function _get_val at 0x7f275aa33c80>>, 'SITE_NAME': '\\xe6\\x92\\xb8\\xe8\\xae\\xaf\\xe5\\x85\\x88\\xe7\\x94\\x9f', 'user': <SimpleLazyObject: <function <lambda> at 0x7f275aa33a28>>, 'SITE_DECS': '\\xe4\\xb8\\x96\\xe7\\x95\\x8c\\xe4\\xb8\\x8a\\xe6\\x9c\\xac\\xe6\\xb2\\xa1\\xe6\\x9c\\x89\\xe5\\x9f\\xba\\xe4\\xbd\\xac\\xef\\xbc\\x8c\\xe8\\xaf\\xb4\\xe7\\x9a\\x84\\xe4\\xba\\xba\\xe5\\xa4\\x9a\\xe4\\xba\\x86\\xef\\xbc\\x8c\\xe4\\xb9\\x9f\\xe5\\xb0\\xb1\\xe6\\x88\\x90\\xe4\\xba\\x86\\xe5\\x9f\\xba\\xe4\\xbd\\xac', 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x7f275aa064d0>, 'DEFAULT_MESSAGE_LEVELS': {'DEBUG': 10, 'INFO': 20, 'WARNING': 30, 'SUCCESS': 25, 'ERROR': 40}, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x7f275a1c1b50>, u'request': <WSGIRequest: GET '/'>}, {}, {'request': <WSGIRequest: GET '/'>}]"
2017-03-13 12:22:46,647 [Thread-27:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET / HTTP/1.1" 200 14898
2017-03-13 12:22:46,684 [Thread-28:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/css/base.css HTTP/1.1" 304 0
2017-03-13 12:22:46,685 [Thread-29:139807009523456] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/css/index.css HTTP/1.1" 304 0
2017-03-13 12:22:46,688 [Thread-30:139807009523456] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/js/jquery.min.js HTTP/1.1" 304 0
2017-03-13 12:22:46,690 [Thread-32:139807009523456] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/js/nav.js HTTP/1.1" 304 0
2017-03-13 12:22:46,692 [Thread-31:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/js/sliders.js HTTP/1.1" 304 0
2017-03-13 12:22:46,746 [Thread-33:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/a1.jpg HTTP/1.1" 304 0
2017-03-13 12:22:46,752 [Thread-34:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/a2.jpg HTTP/1.1" 304 0
2017-03-13 12:22:46,758 [Thread-35:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/a3.jpg HTTP/1.1" 304 0
2017-03-13 12:22:46,769 [Thread-36:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/a4.jpg HTTP/1.1" 304 0
2017-03-13 12:22:46,773 [Thread-37:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/newsbg01.png HTTP/1.1" 304 0
2017-03-13 12:22:46,777 [Thread-38:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/newsbg02.png HTTP/1.1" 304 0
2017-03-13 12:22:46,791 [Thread-39:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/newsbg04.png HTTP/1.1" 304 0
2017-03-13 12:22:46,797 [Thread-40:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/newsbg03.png HTTP/1.1" 304 0
2017-03-13 12:22:46,812 [Thread-41:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_05_1.jpg HTTP/1.1" 304 0
2017-03-13 12:22:46,816 [Thread-42:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_06_1.jpg HTTP/1.1" 304 0
2017-03-13 12:22:46,819 [Thread-43:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_04_1.jpg HTTP/1.1" 304 0
2017-03-13 12:22:46,822 [Thread-44:139807009523456] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_07_1.jpg HTTP/1.1" 304 0
2017-03-13 12:22:46,824 [Thread-45:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ph2.png HTTP/1.1" 304 0
2017-03-13 12:22:46,835 [Thread-46:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/datepng.png HTTP/1.1" 304 0
2017-03-13 12:22:46,837 [Thread-47:139807009523456] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/icons.png HTTP/1.1" 304 0
2017-03-13 12:22:46,927 [Thread-48:139807009523456] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/slide-bg.png HTTP/1.1" 304 0
2017-03-13 12:22:46,931 [Thread-49:139807009523456] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/silde-nav.png HTTP/1.1" 304 0
2017-03-13 12:23:04,480 [Thread-51:139807009523456] [django.template:929] [base:_resolve_lookup] [DEBUG]- Exception while resolving variable 'SITE_DESC' in template 'index.html'.
Traceback (most recent call last):
  File "/home/cc/project_virtualenv/blog_project_venv/local/lib/python2.7/site-packages/django/template/base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [SITE_DESC] in u"[{'False': False, 'None': None, 'True': True}, {u'csrf_token': <SimpleLazyObject: <function _get_val at 0x7f275aa33d70>>, 'SITE_NAME': '\\xe6\\x92\\xb8\\xe8\\xae\\xaf\\xe5\\x85\\x88\\xe7\\x94\\x9f', 'user': <SimpleLazyObject: <function <lambda> at 0x7f275aa33c80>>, 'SITE_DECS': '\\xe4\\xb8\\x96\\xe7\\x95\\x8c\\xe4\\xb8\\x8a\\xe6\\x9c\\xac\\xe6\\xb2\\xa1\\xe6\\x9c\\x89\\xe5\\x9f\\xba\\xe4\\xbd\\xac\\xef\\xbc\\x8c\\xe8\\xaf\\xb4\\xe7\\x9a\\x84\\xe4\\xba\\xba\\xe5\\xa4\\x9a\\xe4\\xba\\x86\\xef\\xbc\\x8c\\xe4\\xb9\\x9f\\xe5\\xb0\\xb1\\xe6\\x88\\x90\\xe4\\xba\\x86\\xe5\\x9f\\xba\\xe4\\xbd\\xac', 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x7f275a1d2e90>, 'DEFAULT_MESSAGE_LEVELS': {'DEBUG': 10, 'INFO': 20, 'WARNING': 30, 'SUCCESS': 25, 'ERROR': 40}, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x7f275a1ca110>, u'request': <WSGIRequest: GET '/'>}, {}, {'request': <WSGIRequest: GET '/'>}]"
2017-03-13 12:23:04,482 [Thread-51:139807009523456] [django.server:131] [basehttp:log_message] [INFO]- "GET / HTTP/1.1" 200 14898
2017-03-13 12:23:04,517 [Thread-52:139807009523456] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/css/base.css HTTP/1.1" 304 0
2017-03-13 12:23:04,519 [Thread-53:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/css/index.css HTTP/1.1" 304 0
2017-03-13 12:23:04,521 [Thread-54:139807009523456] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/js/jquery.min.js HTTP/1.1" 304 0
2017-03-13 12:23:04,528 [Thread-55:139807009523456] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/js/sliders.js HTTP/1.1" 304 0
2017-03-13 12:23:04,530 [Thread-56:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/js/nav.js HTTP/1.1" 304 0
2017-03-13 12:23:04,607 [Thread-57:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/a1.jpg HTTP/1.1" 304 0
2017-03-13 12:23:04,612 [Thread-58:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/a2.jpg HTTP/1.1" 304 0
2017-03-13 12:23:04,616 [Thread-59:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/a3.jpg HTTP/1.1" 304 0
2017-03-13 12:23:04,620 [Thread-60:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/a4.jpg HTTP/1.1" 304 0
2017-03-13 12:23:04,651 [Thread-61:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/newsbg01.png HTTP/1.1" 304 0
2017-03-13 12:23:04,654 [Thread-62:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/newsbg02.png HTTP/1.1" 304 0
2017-03-13 12:23:04,657 [Thread-63:139807009523456] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/newsbg04.png HTTP/1.1" 304 0
2017-03-13 12:23:04,663 [Thread-65:139807000868608] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_05_1.jpg HTTP/1.1" 304 0
2017-03-13 12:23:04,671 [Thread-66:139806992213760] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/newsbg03.png HTTP/1.1" 304 0
2017-03-13 12:23:04,673 [Thread-64:139807009523456] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_06_1.jpg HTTP/1.1" 304 0
2017-03-13 12:23:04,674 [Thread-68:139806992213760] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_07_1.jpg HTTP/1.1" 304 0
2017-03-13 12:23:04,675 [Thread-67:139806983821056] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_04_1.jpg HTTP/1.1" 304 0
2017-03-13 12:23:04,678 [Thread-69:139806983821056] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ph2.png HTTP/1.1" 304 0
2017-03-13 12:23:04,680 [Thread-70:139806992213760] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/datepng.png HTTP/1.1" 304 0
2017-03-13 12:23:04,684 [Thread-71:139806992213760] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/icons.png HTTP/1.1" 304 0
2017-03-13 12:23:04,703 [Thread-72:139806992213760] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/slide-bg.png HTTP/1.1" 304 0
2017-03-13 12:23:04,707 [Thread-73:139806992213760] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/silde-nav.png HTTP/1.1" 304 0
2017-03-13 12:23:15,984 [Dummy-1:140217437632256] [django.db.backends:90] [utils:execute] [DEBUG]- (0.001) 
            SELECT name, type FROM sqlite_master
            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'
            ORDER BY name; args=None
2017-03-13 12:23:15,991 [Dummy-1:140217437632256] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
2017-03-13 12:23:16,644 [Thread-2:140217425757952] [django.template:929] [base:_resolve_lookup] [DEBUG]- Exception while resolving variable 'SITE_DESC' in template 'index.html'.
Traceback (most recent call last):
  File "/home/cc/project_virtualenv/blog_project_venv/local/lib/python2.7/site-packages/django/template/base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [SITE_DESC] in u"[{'False': False, 'None': None, 'True': True}, {u'csrf_token': <SimpleLazyObject: <function _get_val at 0x7f86e9ff4aa0>>, 'SITE_NAME': '\\xe6\\x92\\xb8\\xe8\\xae\\xaf\\xe5\\x85\\x88\\xe7\\x94\\x9f', 'user': <SimpleLazyObject: <function <lambda> at 0x7f86e9fd6488>>, 'SITE_DECS': '\\xe4\\xb8\\x96\\xe7\\x95\\x8c\\xe4\\xb8\\x8a\\xe6\\x9c\\xac\\xe6\\xb2\\xa1\\xe6\\x9c\\x89\\xe5\\x9f\\xba\\xe4\\xbd\\xac\\xef\\xbc\\x8c\\xe8\\xaf\\xb4\\xe7\\x9a\\x84\\xe4\\xba\\xba\\xe5\\xa4\\x9a\\xe4\\xba\\x86\\xef\\xbc\\x8c\\xe4\\xb9\\x9f\\xe5\\xb0\\xb1\\xe6\\x88\\x90\\xe4\\xba\\x86\\xe5\\x9f\\xba\\xe4\\xbd\\xac', 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x7f86e9578750>, 'DEFAULT_MESSAGE_LEVELS': {'DEBUG': 10, 'INFO': 20, 'WARNING': 30, 'SUCCESS': 25, 'ERROR': 40}, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x7f86e9fd3950>, u'request': <WSGIRequest: GET '/'>}, {}, {'request': <WSGIRequest: GET '/'>}]"
2017-03-13 12:23:16,647 [Thread-2:140217425757952] [django.server:131] [basehttp:log_message] [INFO]- "GET / HTTP/1.1" 200 14898
2017-03-13 12:23:16,681 [Thread-4:140217417103104] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/css/index.css HTTP/1.1" 304 0
2017-03-13 12:23:16,682 [Thread-3:140217425757952] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/css/base.css HTTP/1.1" 304 0
2017-03-13 12:23:16,686 [Thread-5:140217425757952] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/js/jquery.min.js HTTP/1.1" 304 0
2017-03-13 12:23:16,687 [Thread-6:140217417103104] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/js/sliders.js HTTP/1.1" 304 0
2017-03-13 12:23:16,692 [Thread-7:140217417103104] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/js/nav.js HTTP/1.1" 304 0
2017-03-13 12:23:16,748 [Thread-8:140217417103104] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/a1.jpg HTTP/1.1" 304 0
2017-03-13 12:23:16,752 [Thread-9:140217417103104] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/a2.jpg HTTP/1.1" 304 0
2017-03-13 12:23:16,757 [Thread-10:140217417103104] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/a3.jpg HTTP/1.1" 304 0
2017-03-13 12:23:16,765 [Thread-11:140217417103104] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/a4.jpg HTTP/1.1" 304 0
2017-03-13 12:23:16,772 [Thread-12:140217417103104] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/newsbg01.png HTTP/1.1" 304 0
2017-03-13 12:23:16,773 [Thread-13:140217425757952] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/newsbg02.png HTTP/1.1" 304 0
2017-03-13 12:23:16,778 [Thread-14:140217425757952] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/newsbg04.png HTTP/1.1" 304 0
2017-03-13 12:23:16,785 [Thread-15:140217425757952] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/newsbg03.png HTTP/1.1" 304 0
2017-03-13 12:23:16,814 [Thread-16:140217425757952] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_05_1.jpg HTTP/1.1" 304 0
2017-03-13 12:23:16,821 [Thread-17:140217425757952] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_06_1.jpg HTTP/1.1" 304 0
2017-03-13 12:23:16,830 [Thread-18:140217425757952] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_04_1.jpg HTTP/1.1" 304 0
2017-03-13 12:23:16,836 [Thread-19:140217425757952] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_07_1.jpg HTTP/1.1" 304 0
2017-03-13 12:23:16,841 [Thread-20:140217425757952] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ph2.png HTTP/1.1" 304 0
2017-03-13 12:23:16,847 [Thread-21:140217425757952] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/datepng.png HTTP/1.1" 304 0
2017-03-13 12:23:16,851 [Thread-22:140217425757952] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/icons.png HTTP/1.1" 304 0
2017-03-13 12:23:16,888 [Thread-23:140217425757952] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/slide-bg.png HTTP/1.1" 304 0
2017-03-13 12:23:16,900 [Thread-24:140217425757952] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/silde-nav.png HTTP/1.1" 304 0
2017-03-13 12:23:24,445 [Dummy-1:140715597559552] [django.db.backends:90] [utils:execute] [DEBUG]- (0.001) 
            SELECT name, type FROM sqlite_master
            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'
            ORDER BY name; args=None
2017-03-13 12:23:24,449 [Dummy-1:140715597559552] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
2017-03-13 12:23:29,631 [Thread-2:140715585685248] [django.template:929] [base:_resolve_lookup] [DEBUG]- Exception while resolving variable 'SITE_DESC' in template 'index.html'.
Traceback (most recent call last):
  File "/home/cc/project_virtualenv/blog_project_venv/local/lib/python2.7/site-packages/django/template/base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [SITE_DESC] in u"[{'False': False, 'None': None, 'True': True}, {u'csrf_token': <SimpleLazyObject: <function _get_val at 0x7ffae5fbcb18>>, 'SITE_NAME': '\\xe6\\x92\\xb8\\xe8\\xae\\xaf\\xe5\\x85\\x88\\xe7\\x94\\x9f', 'user': <SimpleLazyObject: <function <lambda> at 0x7ffae6a15500>>, 'SITE_DECS': '\\xe4\\xb8\\x96\\xe7\\x95\\x8c\\xe4\\xb8\\x8a\\xe6\\x9c\\xac\\xe6\\xb2\\xa1\\xe6\\x9c\\x89\\xe5\\x9f\\xba\\xe4\\xbd\\xac\\xef\\xbc\\x8c\\xe8\\xaf\\xb4\\xe7\\x9a\\x84\\xe4\\xba\\xba\\xe5\\xa4\\x9a\\xe4\\xba\\x86\\xef\\xbc\\x8c\\xe4\\xb9\\x9f\\xe5\\xb0\\xb1\\xe6\\x88\\x90\\xe4\\xba\\x86\\xe5\\x9f\\xba\\xe4\\xbd\\xac', 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x7ffae5fce710>, 'DEFAULT_MESSAGE_LEVELS': {'DEBUG': 10, 'INFO': 20, 'WARNING': 30, 'SUCCESS': 25, 'ERROR': 40}, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x7ffae6a12910>, u'request': <WSGIRequest: GET '/'>}, {}, {'request': <WSGIRequest: GET '/'>}]"
2017-03-13 12:23:29,637 [Thread-2:140715585685248] [django.server:131] [basehttp:log_message] [INFO]- "GET / HTTP/1.1" 200 14898
2017-03-13 12:23:32,137 [Thread-3:140715585685248] [django.template:929] [base:_resolve_lookup] [DEBUG]- Exception while resolving variable 'SITE_DESC' in template 'index.html'.
Traceback (most recent call last):
  File "/home/cc/project_virtualenv/blog_project_venv/local/lib/python2.7/site-packages/django/template/base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [SITE_DESC] in u"[{'False': False, 'None': None, 'True': True}, {u'csrf_token': <SimpleLazyObject: <function _get_val at 0x7ffae5ff4b90>>, 'SITE_NAME': '\\xe6\\x92\\xb8\\xe8\\xae\\xaf\\xe5\\x85\\x88\\xe7\\x94\\x9f', 'user': <SimpleLazyObject: <function <lambda> at 0x7ffae5ff49b0>>, 'SITE_DECS': '\\xe4\\xb8\\x96\\xe7\\x95\\x8c\\xe4\\xb8\\x8a\\xe6\\x9c\\xac\\xe6\\xb2\\xa1\\xe6\\x9c\\x89\\xe5\\x9f\\xba\\xe4\\xbd\\xac\\xef\\xbc\\x8c\\xe8\\xaf\\xb4\\xe7\\x9a\\x84\\xe4\\xba\\xba\\xe5\\xa4\\x9a\\xe4\\xba\\x86\\xef\\xbc\\x8c\\xe4\\xb9\\x9f\\xe5\\xb0\\xb1\\xe6\\x88\\x90\\xe4\\xba\\x86\\xe5\\x9f\\xba\\xe4\\xbd\\xac', 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x7ffae5f83190>, 'DEFAULT_MESSAGE_LEVELS': {'DEBUG': 10, 'INFO': 20, 'WARNING': 30, 'SUCCESS': 25, 'ERROR': 40}, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x7ffae5fc84d0>, u'request': <WSGIRequest: GET '/'>}, {}, {'request': <WSGIRequest: GET '/'>}]"
2017-03-13 12:23:32,139 [Thread-3:140715585685248] [django.server:131] [basehttp:log_message] [INFO]- "GET / HTTP/1.1" 200 14898
2017-03-13 12:23:32,175 [Thread-4:140715585685248] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/css/base.css HTTP/1.1" 304 0
2017-03-13 12:23:32,179 [Thread-7:140715568375552] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/js/nav.js HTTP/1.1" 304 0
2017-03-13 12:23:32,193 [Thread-9:140715568375552] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/css/index.css HTTP/1.1" 304 0
2017-03-13 12:23:32,196 [Thread-10:140715208210176] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/js/jquery.min.js HTTP/1.1" 304 0
2017-03-13 12:23:32,198 [Thread-8:140715559982848] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/js/sliders.js HTTP/1.1" 304 0
2017-03-13 12:23:32,247 [Thread-6:140715576768256] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/a1.jpg HTTP/1.1" 304 0
2017-03-13 12:23:32,248 [Thread-5:140715585685248] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/a2.jpg HTTP/1.1" 304 0
2017-03-13 12:23:32,253 [Thread-11:140715585685248] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/a3.jpg HTTP/1.1" 304 0
2017-03-13 12:23:32,260 [Thread-12:140715585685248] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/a4.jpg HTTP/1.1" 304 0
2017-03-13 12:23:32,269 [Thread-13:140715585685248] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/newsbg01.png HTTP/1.1" 304 0
2017-03-13 12:23:32,272 [Thread-14:140715585685248] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/newsbg02.png HTTP/1.1" 304 0
2017-03-13 12:23:32,276 [Thread-15:140715576768256] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/newsbg04.png HTTP/1.1" 304 0
2017-03-13 12:23:32,283 [Thread-16:140715576768256] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/newsbg03.png HTTP/1.1" 304 0
2017-03-13 12:23:32,289 [Thread-17:140715576768256] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_05_1.jpg HTTP/1.1" 304 0
2017-03-13 12:23:32,293 [Thread-18:140715576768256] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_06_1.jpg HTTP/1.1" 304 0
2017-03-13 12:23:32,296 [Thread-19:140715585685248] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_04_1.jpg HTTP/1.1" 304 0
2017-03-13 12:23:32,302 [Thread-20:140715585685248] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_07_1.jpg HTTP/1.1" 304 0
2017-03-13 12:23:32,306 [Thread-23:140715208210176] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ph2.png HTTP/1.1" 304 0
2017-03-13 12:23:32,351 [Thread-22:140715559982848] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/datepng.png HTTP/1.1" 304 0
2017-03-13 12:23:32,356 [Thread-21:140715576768256] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/icons.png HTTP/1.1" 304 0
2017-03-13 12:23:32,432 [Thread-24:140715576768256] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/slide-bg.png HTTP/1.1" 304 0
2017-03-13 12:23:32,436 [Thread-25:140715576768256] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/silde-nav.png HTTP/1.1" 304 0
2017-03-13 12:23:41,980 [Thread-28:140715576768256] [django.template:929] [base:_resolve_lookup] [DEBUG]- Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/cc/project_virtualenv/blog_project_venv/local/lib/python2.7/site-packages/django/template/base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [name] in u'<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>'
2017-03-13 12:23:41,984 [Thread-28:140715576768256] [django.server:131] [basehttp:log_message] [WARNING]- "GET /api/plant/test/ HTTP/1.1" 404 2081
2017-03-13 12:23:44,753 [Thread-29:140715576768256] [django.template:929] [base:_resolve_lookup] [DEBUG]- Exception while resolving variable 'SITE_DESC' in template 'index.html'.
Traceback (most recent call last):
  File "/home/cc/project_virtualenv/blog_project_venv/local/lib/python2.7/site-packages/django/template/base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [SITE_DESC] in u"[{'False': False, 'None': None, 'True': True}, {u'csrf_token': <SimpleLazyObject: <function _get_val at 0x7ffae5ff4c80>>, 'SITE_NAME': '\\xe6\\x92\\xb8\\xe8\\xae\\xaf\\xe5\\x85\\x88\\xe7\\x94\\x9f', 'user': <SimpleLazyObject: <function <lambda> at 0x7ffae5ff49b0>>, 'SITE_DECS': '\\xe4\\xb8\\x96\\xe7\\x95\\x8c\\xe4\\xb8\\x8a\\xe6\\x9c\\xac\\xe6\\xb2\\xa1\\xe6\\x9c\\x89\\xe5\\x9f\\xba\\xe4\\xbd\\xac\\xef\\xbc\\x8c\\xe8\\xaf\\xb4\\xe7\\x9a\\x84\\xe4\\xba\\xba\\xe5\\xa4\\x9a\\xe4\\xba\\x86\\xef\\xbc\\x8c\\xe4\\xb9\\x9f\\xe5\\xb0\\xb1\\xe6\\x88\\x90\\xe4\\xba\\x86\\xe5\\x9f\\xba\\xe4\\xbd\\xac', 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x7ffae5f91910>, 'DEFAULT_MESSAGE_LEVELS': {'DEBUG': 10, 'INFO': 20, 'WARNING': 30, 'SUCCESS': 25, 'ERROR': 40}, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x7ffae5f83510>, u'request': <WSGIRequest: GET '/'>}, {}, {'request': <WSGIRequest: GET '/'>}]"
2017-03-13 12:23:44,756 [Thread-29:140715576768256] [django.server:131] [basehttp:log_message] [INFO]- "GET / HTTP/1.1" 200 14898
2017-03-13 12:24:12,631 [Dummy-1:139923694860032] [django.db.backends:90] [utils:execute] [DEBUG]- (0.009) 
            SELECT name, type FROM sqlite_master
            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'
            ORDER BY name; args=None
2017-03-13 12:24:12,636 [Dummy-1:139923694860032] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
2017-03-13 12:24:20,696 [Thread-2:139923682985728] [django.template:929] [base:_resolve_lookup] [DEBUG]- Exception while resolving variable 'SITE_DESC' in template 'index.html'.
Traceback (most recent call last):
  File "/home/cc/project_virtualenv/blog_project_venv/local/lib/python2.7/site-packages/django/template/base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [SITE_DESC] in u"[{'False': False, 'None': None, 'True': True}, {u'csrf_token': <SimpleLazyObject: <function _get_val at 0x7f4284e78b18>>, 'SITE_NAME': '\\xe6\\x92\\xb8\\xe8\\xae\\xaf\\xe5\\x85\\x88\\xe7\\x94\\x9f', 'user': <SimpleLazyObject: <function <lambda> at 0x7f42858db500>>, 'SITE_DECS': '\\xe4\\xb8\\x96\\xe7\\x95\\x8c\\xe4\\xb8\\x8a\\xe6\\x9c\\xac\\xe6\\xb2\\xa1\\xe6\\x9c\\x89\\xe5\\x9f\\xba\\xe4\\xbd\\xac\\xef\\xbc\\x8c\\xe8\\xaf\\xb4\\xe7\\x9a\\x84\\xe4\\xba\\xba\\xe5\\xa4\\x9a\\xe4\\xba\\x86\\xef\\xbc\\x8c\\xe4\\xb9\\x9f\\xe5\\xb0\\xb1\\xe6\\x88\\x90\\xe4\\xba\\x86\\xe5\\x9f\\xba\\xe4\\xbd\\xac', 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x7f4284e8a750>, 'DEFAULT_MESSAGE_LEVELS': {'DEBUG': 10, 'INFO': 20, 'WARNING': 30, 'SUCCESS': 25, 'ERROR': 40}, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x7f42858d9950>, u'request': <WSGIRequest: GET '/'>}, {}, {'request': <WSGIRequest: GET '/'>}]"
2017-03-13 12:24:20,714 [Thread-2:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET / HTTP/1.1" 200 14898
2017-03-13 12:24:20,820 [Thread-3:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/css/base.css HTTP/1.1" 304 0
2017-03-13 12:24:20,823 [Thread-4:139923674068736] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/css/index.css HTTP/1.1" 304 0
2017-03-13 12:24:20,825 [Thread-5:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/js/jquery.min.js HTTP/1.1" 304 0
2017-03-13 12:24:20,828 [Thread-6:139923674068736] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/js/sliders.js HTTP/1.1" 304 0
2017-03-13 12:24:20,835 [Thread-7:139923674068736] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/js/nav.js HTTP/1.1" 304 0
2017-03-13 12:24:20,954 [Thread-8:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/a1.jpg HTTP/1.1" 304 0
2017-03-13 12:24:20,957 [Thread-9:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/a2.jpg HTTP/1.1" 304 0
2017-03-13 12:24:20,971 [Thread-10:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/a4.jpg HTTP/1.1" 304 0
2017-03-13 12:24:21,008 [Thread-11:139923674068736] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/a3.jpg HTTP/1.1" 304 0
2017-03-13 12:24:21,055 [Thread-13:139923457832704] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/newsbg01.png HTTP/1.1" 304 0
2017-03-13 12:24:21,059 [Thread-12:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/newsbg02.png HTTP/1.1" 304 0
2017-03-13 12:24:21,061 [Thread-14:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/newsbg04.png HTTP/1.1" 304 0
2017-03-13 12:24:21,080 [Thread-15:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/newsbg03.png HTTP/1.1" 304 0
2017-03-13 12:24:21,090 [Thread-16:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_05_1.jpg HTTP/1.1" 304 0
2017-03-13 12:24:21,106 [Thread-17:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_04_1.jpg HTTP/1.1" 304 0
2017-03-13 12:24:21,109 [Thread-18:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_06_1.jpg HTTP/1.1" 304 0
2017-03-13 12:24:21,116 [Thread-20:139923457832704] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_07_1.jpg HTTP/1.1" 304 0
2017-03-13 12:24:21,127 [Thread-19:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ph2.png HTTP/1.1" 304 0
2017-03-13 12:24:21,142 [Thread-21:139923674068736] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/datepng.png HTTP/1.1" 304 0
2017-03-13 12:24:21,165 [Thread-22:139923674068736] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/icons.png HTTP/1.1" 304 0
2017-03-13 12:24:21,296 [Thread-24:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/slide-bg.png HTTP/1.1" 304 0
2017-03-13 12:24:21,300 [Thread-23:139923674068736] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/silde-nav.png HTTP/1.1" 304 0
2017-03-13 12:24:25,440 [Thread-25:139923682985728] [django.server:131] [basehttp:log_message] [WARNING]- "GET /static/index.html HTTP/1.1" 404 1643
2017-03-13 12:25:18,221 [Thread-27:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET / HTTP/1.1" 200 14961
2017-03-13 12:25:18,510 [Thread-28:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/css/base.css HTTP/1.1" 304 0
2017-03-13 12:25:18,538 [Thread-29:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/css/index.css HTTP/1.1" 304 0
2017-03-13 12:25:18,555 [Thread-30:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/js/jquery.min.js HTTP/1.1" 304 0
2017-03-13 12:25:18,565 [Thread-31:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/js/sliders.js HTTP/1.1" 304 0
2017-03-13 12:25:18,576 [Thread-32:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/js/nav.js HTTP/1.1" 304 0
2017-03-13 12:25:18,855 [Thread-33:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/a1.jpg HTTP/1.1" 304 0
2017-03-13 12:25:18,872 [Thread-34:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/a2.jpg HTTP/1.1" 304 0
2017-03-13 12:25:18,888 [Thread-35:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/a3.jpg HTTP/1.1" 304 0
2017-03-13 12:25:18,901 [Thread-36:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/a4.jpg HTTP/1.1" 304 0
2017-03-13 12:25:18,938 [Thread-37:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/newsbg01.png HTTP/1.1" 304 0
2017-03-13 12:25:18,967 [Thread-38:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/newsbg02.png HTTP/1.1" 304 0
2017-03-13 12:25:18,994 [Thread-39:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/newsbg04.png HTTP/1.1" 304 0
2017-03-13 12:25:19,001 [Thread-40:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/newsbg03.png HTTP/1.1" 304 0
2017-03-13 12:25:19,035 [Thread-41:139923682985728] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_05_1.jpg HTTP/1.1" 304 0
2017-03-13 12:25:19,044 [Thread-42:139923674068736] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_06_1.jpg HTTP/1.1" 304 0
2017-03-13 12:25:19,055 [Thread-43:139923674068736] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_04_1.jpg HTTP/1.1" 304 0
2017-03-13 12:25:19,072 [Thread-44:139923674068736] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_07_1.jpg HTTP/1.1" 304 0
2017-03-13 12:25:19,096 [Thread-45:139923674068736] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ph2.png HTTP/1.1" 304 0
2017-03-13 12:25:19,106 [Thread-46:139923674068736] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/datepng.png HTTP/1.1" 304 0
2017-03-13 12:25:19,126 [Thread-47:139923674068736] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/icons.png HTTP/1.1" 304 0
2017-03-13 12:25:19,440 [Thread-48:139923674068736] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/slide-bg.png HTTP/1.1" 304 0
2017-03-13 12:25:19,457 [Thread-49:139923674068736] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/silde-nav.png HTTP/1.1" 304 0
2017-03-13 13:01:23,393 [Dummy-1:140025579788032] [django.db.backends:90] [utils:execute] [DEBUG]- (0.009) 
            SELECT name, type FROM sqlite_master
            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'
            ORDER BY name; args=None
2017-03-13 13:01:23,468 [Dummy-1:140025579788032] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
2017-03-13 13:02:10,361 [Thread-2:140025567913728] [django.server:131] [basehttp:log_message] [INFO]- "GET / HTTP/1.1" 200 14961
2017-03-13 13:02:10,797 [Thread-3:140025559521024] [django.template:929] [base:_resolve_lookup] [DEBUG]- Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/cc/project_virtualenv/blog_project_venv/local/lib/python2.7/site-packages/django/template/base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [name] in u'<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>'
2017-03-13 13:02:10,822 [Thread-3:140025559521024] [django.server:131] [basehttp:log_message] [WARNING]- "GET /favicon.ico HTTP/1.1" 404 2069
2017-03-13 13:03:33,474 [Dummy-1:140690871727872] [django.db.backends:90] [utils:execute] [DEBUG]- (0.001) 
            SELECT name, type FROM sqlite_master
            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'
            ORDER BY name; args=None
2017-03-13 13:03:33,478 [Dummy-1:140690871727872] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
2017-03-13 13:38:22,751 [Dummy-1:140248043312896] [django.db.backends:90] [utils:execute] [DEBUG]- (0.009) 
            SELECT name, type FROM sqlite_master
            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'
            ORDER BY name; args=None
2017-03-13 13:38:22,793 [Dummy-1:140248043312896] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
2017-03-14 03:24:38,111 [Dummy-1:140508428973824] [django.db.backends:90] [utils:execute] [DEBUG]- (0.006) 
            SELECT name, type FROM sqlite_master
            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'
            ORDER BY name; args=None
2017-03-14 03:24:38,153 [Dummy-1:140508428973824] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
2017-03-14 03:25:26,268 [MainThread:140481255196416] [django.db.backends:90] [utils:execute] [DEBUG]- (0.001) 
            SELECT name, type FROM sqlite_master
            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'
            ORDER BY name; args=None
2017-03-14 03:25:26,271 [MainThread:140481255196416] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
2017-03-14 03:25:35,905 [MainThread:140053087815424] [django.db.backends:90] [utils:execute] [DEBUG]- (0.001) 
            SELECT name, type FROM sqlite_master
            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'
            ORDER BY name; args=None
2017-03-14 03:25:35,908 [MainThread:140053087815424] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
2017-03-14 03:25:35,913 [MainThread:140053087815424] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) 
            SELECT name, type FROM sqlite_master
            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'
            ORDER BY name; args=None
2017-03-14 03:25:35,914 [MainThread:140053087815424] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
2017-03-14 03:31:09,939 [MainThread:139883728512768] [django.db.backends:90] [utils:execute] [DEBUG]- (0.001) 
            SELECT name, type FROM sqlite_master
            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'
            ORDER BY name; args=None
2017-03-14 03:31:09,941 [MainThread:139883728512768] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
2017-03-14 03:32:42,743 [Dummy-1:140081564378880] [django.db.backends:90] [utils:execute] [DEBUG]- (0.001) 
            SELECT name, type FROM sqlite_master
            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'
            ORDER BY name; args=None
2017-03-14 03:32:42,750 [Dummy-1:140081564378880] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
2017-03-14 03:33:21,897 [Thread-2:140081535985408] [django.server:131] [basehttp:log_message] [INFO]- "GET / HTTP/1.1" 200 14961
2017-03-14 03:33:36,254 [Thread-8:140081535985408] [django.server:131] [basehttp:log_message] [INFO]- "GET / HTTP/1.1" 200 14961
2017-03-14 03:33:49,796 [Thread-9:140081535985408] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/images/ico_07.png HTTP/1.1" 200 2692
2017-03-14 04:46:03,550 [MainThread:139903796143872] [django.db.backends:90] [utils:execute] [DEBUG]- (0.005) 
            SELECT name, type FROM sqlite_master
            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'
            ORDER BY name; args=None
2017-03-14 04:46:03,582 [MainThread:139903796143872] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
2017-03-14 04:54:47,179 [MainThread:139831442122496] [django.db.backends:90] [utils:execute] [DEBUG]- (0.001) 
            SELECT name, type FROM sqlite_master
            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'
            ORDER BY name; args=None
2017-03-14 04:54:47,181 [MainThread:139831442122496] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
2017-03-14 04:55:01,612 [MainThread:139633976338176] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) 
            SELECT name, type FROM sqlite_master
            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'
            ORDER BY name; args=None
2017-03-14 04:55:01,615 [MainThread:139633976338176] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
2017-03-14 04:55:01,616 [MainThread:139633976338176] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) 
            SELECT name, type FROM sqlite_master
            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'
            ORDER BY name; args=None
2017-03-14 04:55:01,620 [MainThread:139633976338176] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
2017-03-15 00:24:16,107 [MainThread:139675929097984] [django.db.backends:90] [utils:execute] [DEBUG]- (0.006) 
            SELECT name, type FROM sqlite_master
            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'
            ORDER BY name; args=None
2017-03-15 00:24:16,138 [MainThread:139675929097984] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
2017-03-15 00:24:16,139 [MainThread:139675929097984] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) 
            SELECT name, type FROM sqlite_master
            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'
            ORDER BY name; args=None
2017-03-15 00:24:16,140 [MainThread:139675929097984] [django.db.backends:90] [utils:execute] [DEBUG]- (0.000) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
